example_scripts/SasView_example.py

```diff
@@ -0,0 +1,87 @@
+from sasmodels.core import load_model
```

Comment:
What is the purpose of this file and how does it differ from example_runScripts_SasView.py